modules/services/swayidle.nix

```diff
@@ -73,7 +73,7 @@ in {
       default = [ ];
       example = literalExpression ''
         [
-          { timeout = 60; command = "swaylock -fF"; }
+          { timeout = 60; command = "${pkgs.swaylock}/bin/swaylock -fF"; }
```

Comment:
```suggestion
          { timeout = 60; command = ''"''${pkgs.swaylock}/bin/swaylock -fF"''; }
```

Reply:
I think you want the parameters (`-fF`) on the outside of the quotes